FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c

```diff
@@ -54,7 +54,7 @@
 #include "metrics_collector.h"
 /*-----------------------------------------------------------*/
 
-eMetricsCollectorStatus xGetNetworkStats( NetworkStats_t * pxOutNetworkStats )
+eMetricsCollectorStatus eGetNetworkStats( NetworkStats_t * pxOutNetworkStats )
```

Comment:
I think it is better to define the eMetricsCollectorStatus as a user defined type. Is there a reason why you kept it this way?